include/output.hpp

```diff
@@ -19,13 +19,19 @@ class Output{
   int output_frequency;   /* Output frequency in time domain: write output at every <num> time step. */
   std::vector<std::vector<std::string> > outputstr; // List of outputs for each oscillator
 
-  bool writefullstate;  /* Flag to determin if full state vector should be written to file */
+  bool writeFullState;  /* Flag to determine if full state vector should be written to file */
   FILE *ufile;          /* File for writing real part of solution vector */
   FILE *vfile;          /* File for writing imaginary part of solution vector */
-  std::vector<FILE *>expectedfile;    /* Files for writing the evolution of the expected energy levels per oscillator */
-  FILE *expectedfile_comp;    /* File for writing the evolution of the expected energy level of the composite system */
-  FILE *populationfile_comp;    /* File for writing the evolution of the population of the composite system */
+
+  std::vector<bool> writeExpectedEnergy; ///< Flag to determine if the evolution of the expected energy per oscillator should be written to files.
+  bool writeExpectedEnergy_comp; ///< Flag to determine if the evolution of the expected energy of the full composite system should be written to file
+  std::vector<FILE *>expectedfile;    /* Files for writing the evolution of the expected energy per oscillator */
```

Comment:
May be nice to use `///<` but in any case this will probably conflict with the `documentation` branch so can be fixed then.